Dockerfile

```diff
@@ -24,12 +24,19 @@ RUN yum install -y \
   curl tar \
   && yum clean all
 
-# install Oracle JRE
+# configure java runtime
 ENV JAVA_HOME=/opt/java \
   JAVA_VERSION_MAJOR=8 \
   JAVA_VERSION_MINOR=102 \
   JAVA_VERSION_BUILD=14
 
+# configure nexus runtime
+ENV NEXUS_VERSION=3.1.0-04 \
+  NEXUS_HOME=/opt/sonatype/nexus \
+  NEXUS_DATA=/nexus-data \
```

Comment:
Thoughts on changing this to be /opt/sonatype/nexus/sonatype-work/nexus and forego the following changes to nexus.vmoptions?